src/platform_impl/windows/window.rs

```diff
@@ -621,6 +621,33 @@ impl Window {
         warn!("`Window::set_ime_position` is ignored on Windows")
     }
 
+    #[inline]
+    pub fn request_user_attention(&self, request_type: Option<RequestUserAttentionType>) {
+        let window = self.window.clone();
+
+        self.thread_executor.execute_in_thread(move || unsafe {
+            let (flags, count) = request_type
+                .map(|ty| match ty {
+                    RequestUserAttentionType::Critical => {
+                        (winuser::FLASHW_ALL | winuser::FLASHW_TIMERNOFG, u32::MAX)
```

Comment:
Why is it `u32::MAX` here and `0` in the other cases? Based on the passed flags there should to be no difference in the behavior imo.

Reply:
The difference between `Critical` and `Informational` on Windows is the continuous flashing of the window itself when `Critical`. `Informational` only flashes the taskbar button. The number of times the window flashes is `u32::MAX`.

Reply:
nvm I somehow confused it with `dwTimeout`